python/jax/carfac_bench.py

```diff
@@ -466,4 +466,4 @@ def bench_jax_util_mapped(state: google_benchmark.State):
 
 
 if __name__ == '__main__':
-  google_benchmark.main()
+  google_benchmark.main()
```

Comment:
Please restore the newline at end of file here.

Reply:
done